src/pages/docs/liveobjects/rest-api-usage.mdx

```diff
@@ -39,357 +39,205 @@ The key in the `data` object indicates the type of the value:
+  -u {{API_KEY}}
+  ```
 </Code>
 
-This request is billed per object returned in the result - each object counts as one billable message.
-
 <Code>
-```json
-[
-  {
-    "objectId": "counter:JbZYiHnw0ORAyzzLSQahVik31iBDL_ehJNpTEF3qwg8@1745828651669",
-    "counter": {
-      "data": {
-        "number": 10
-      }
-    }
-  },
-  {
-    "objectId": "counter:iVji62_MW_j4dShuJbr2fmsP2D8MyCs6tFqON9-xAkc@1745828645269",
-    "counter": {
-      "data": {
-        "number": 5
-      }
-    }
-  },
-  {
-    "objectId": "map:ja7cjMUib2LmJKTRdoGAG9pbBYCnkMObAVpmojCOmek@1745828596519",
-    "map": {
-      "entries": {
-        "down": {
-          "data": {
-            "objectId": "counter:JbZYiHnw0ORAyzzLSQahVik31iBDL_ehJNpTEF3qwg8@1745828651669"
-          }
-        },
-        "up": {
-          "data": {
-            "objectId": "counter:iVji62_MW_j4dShuJbr2fmsP2D8MyCs6tFqON9-xAkc@1745828645269"
-          }
-        }
-      }
-    }
-  },
+  ```json
   {
     "objectId": "root",
     "map": {
+      "semantics": "LWW",
       "entries": {
-        "votes": {
+        "poll": {
           "data": {
-            "objectId": "map:ja7cjMUib2LmJKTRdoGAG9pbBYCnkMObAVpmojCOmek@1745828596519"
+            "objectId": "map:fNDMOPlPriqid9SUitqIuP_S-6u5YUYQzxjs39IM68k@1760448542413",
+            "map": {
+              "semantics": "LWW",
+              "entries": {
+                "title": {
+                  "data": {
+                    "string": "First Poll"
+                  }
+                },
+                "votes": {
+                  "data": {
+                    "objectId": "map:qZXBk8xaGqf4kwLSlR7Tj0Eqhd48WDFfb3gTAbj194k@1760448597692",
+                    "map": {
+                      "semantics": "LWW",
+                      "entries": {
+                        "down": {
+                          "data": {
+                            "objectId": "counter:Yj1F_aEX3T2rRkTkra7Aifmlr8PxUWSR3kO3MzxtQto@1760448653393",
+                            "counter": {
+                              "data": {
+                                "number": 5
+                              }
+                            }
+                          }
+                        },
+                        "up": {
+                          "data": {
+                            "objectId": "counter:ibxddWpDjH8R3cvXWWacfe4IVd3DxT_oqkAafhaS68s@1760448646413",
+                            "counter": {
+                              "data": {
+                                "number": 10
+                              }
+                            }
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
           }
         }
       }
     }
   }
-]
-```
+  ```
 </Code>
 
-#### Including metadata <a id="fetching-objects-list-metadata"/>
+Both these requests are billed per object returned.
```

Comment:
```suggestion
Each instance of an [object type](/docs/liveobjects/concepts/objects#object-types) included is the response is counted as one billable message.
```

I think this statement should also appear at the end of the "Get the entire object" section (so that a reader of that specific section has all the information they need, i.e. we do not hide information relating to one section in the content for other sections)